src/main/java/com/Mybatismovies/demo/MovieServiceImpl.java

```diff
@@ -0,0 +1,33 @@
+package com.Mybatismovies.demo;
+
+import org.springframework.stereotype.Service;
+
+import java.time.Year;
+import java.util.List;
+
+@Service
+public class MovieServiceImpl implements MovieService {
+    private MovieMapper movieMapper;
```

Comment:
コンストラクタインジェクションの場合、finalをつけられるというメリットがあります。
つけておきましょう。
```suggestion
    private final MovieMapper movieMapper;
```